R/bs4_book.R

```diff
@@ -0,0 +1,375 @@
+#' HTML book built with bootstrap4.
+#'
+#' @description
+#' This output format is designed to give a clean reading experience.
+#' Main features:
+#'
+#' * A per section-search engine that helps you quickly find what you're
+#'   looking for.
+#'
+#' * A sidebar that includes a within-chapter table of contents that
+#'   dynamically updates so it's easy to keep track of where you are
+#'   within the chapter.
+#'
+#' * Thoughtful typography to make the contents as easy as possible to read,
+#'   regardless of the size of your device. A sticky header gets out of your
+#'   way when reading, but is easily accessible if you need it.
+#'
+#' * In-line footnotes.
+#'
+#' * R syntax highlighting and autolinking by
+#'   [downlit](http://downlit.r-lib.org/) is paired with a accessible
+#'   theme designed by Alison Hill.
+#'
+#' * The ability to customise colours and fonts through
+#'   [bootstraplib](https://rstudio.github.io/bootstraplib)
+#'
+#' @section Limitations:
+#'
+#' * `bs_book4()` is designed specifically for books that use one chapter per
```

Comment:
Just seen this typo while looking at the on-going PR
```suggestion
#' * `bs4_book()` is designed specifically for books that use one chapter per
```

Reply:
Oops 😆